recipe/conda_build_config.yaml

```
@@ -2,3 +2,14 @@ rust_compiler_version:
     - 1.69
 maturin:
     - 1.1
+
+# The pinnings below allow a matrix build with two libprotobuf versions and
+# replace migrators for libprotobuf 1.57 / 1.58.
+libgrpc:
+    - 1.58
+    - 1.56
+libprotobuf:
+    - 4.24.3
+    - 4.23.3
```

Should we keep [the global pinning for `libprotobuf`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/7927b71839489c008649b56902f588cf7caf0c8e/recipe/conda_build_config.yaml#L500-L501 ) if we are overriding `libprotobuf` here?

```suggestion
    - 4.23.4
    - 4.23.3
```

---

The entries in this matrix are:
- Global pinning (4.23.4) plus migrations (which make it 4.24.3)
- Older libprotobuf pinning that we need (4.23.3)